Tasks/ArchiveFilesV2/task.json

```diff
@@ -18,8 +18,8 @@
     "demands": [],
     "version": {
         "Major": 2,
-        "Minor": 151,
-        "Patch": 2
+        "Minor": 152,
```

Comment:
The convention that people use is to make it the number of the sprint the change is made it. I am not sure I entirely agree but that's how it's done :)